### ISSUE TITLE
[Bug] System is not functional

### ISSUE THREAD

**Issue by patham9** [opennars/OpenNARS-4#112]
## Describe the bug

`python3 -m pynars.Console`
leads to message "please install the module by `pip install jstyleson`"
even though that package is already installed.

## To Reproduce

Enter

`pip3 install pynars`

Then enter

`python3 -m pynars.Console
`
## Expected behavior

Running system

## Environment (please complete the following information)

- OS: MacOS 13.4
- Language: Python 3.9.6
- The version in current pip

**Comment by patham9** [comment]
Also:
`ModuleNotFoundError: No module named 'pynars.utils.SparseLUT.sparse_lut'
`

**Comment by bowen-xu** [comment]
@patham9 Please use the "opennars4" branch.

**Comment by bowen-xu** [comment]
@patham9 could you please also set the default branch to be "opennars4"? I think I don't have the authority to change the default branch.

**Comment by MinaGabriel** [comment]
@bowen-xu I am still having the same issue, `ModuleNotFoundError: No module named 'pynars.utils.SparseLUT.sparse_lut'` when I run test_IndexVar.py, for example.

**Comment by bowen-xu** [comment]
@MinaGabriel which branch are you using?

**Comment by MinaGabriel** [comment]
@bowen-xu  dev

**Comment by bowen-xu** [comment]
@MinaGabriel Please use the "opennars4" branch. I should have set `opennars4` as the default branch. But I don't have the authority. Maybe someone (Patrick? @patham9 ) can help to change?

**Comment by MinaGabriel** [comment]
I switched the branch, I am getting this error (I am not sure if there is a module missing from pynars), or it has to do with config.json 🤔
```

(base)  ~/Documents/opennars-projects/OpenNARS-4/Tests/ [opennars4*] python test_IndexVar.py
PyNARS version v0.0.2 pre-alpha
The file `config.json` does not exist.

ModuleNotFoundError: No module named 'pynars.utils.SparseLUT.sparse_lut'
```

**Comment by bowen-xu** [comment]
Did you run through this command?
```
python -m opennars
```

**Comment by MinaGabriel** [comment]
Yes, everything works fine except for the test files for me. It's fine, I think some Python version is missing files or so, I will figure it out. Thank you so much!
